src/main/java/stirling/software/SPDF/controller/api/misc/FakeScanControllerWIP.java

```diff
@@ -173,7 +173,7 @@ public BufferedImage softenEdges(BufferedImage image, int featherRadius) {
 
 
     private void addGaussianNoise(BufferedImage image, double strength) {
-        Random rand = new Random();
```

Comment:
@pixeebot  This added SecureRandom without the import causing compile errors

Reply:
pixeebot maintainer here 👋 

Thanks for the tip! But, when I look at the commit, it looks like `SecureRandom` was already there in the imports. Was there some git magic to change the history? I can't find the check that failed because of this either. Would love to help but not sure what to make of this at the moment.

Reply:
Very odd I see the same.. I can only conclude magic 😂 
Thanks for checking

Reply:
Anytime!